src/frontend/src/utils/status.ts

```diff
@@ -293,6 +298,7 @@ export const statusList = (pia: IPiaForm | null): StatusList => {
       Pages: {
         review: {
           accessControl: true,
+          viewProgramAreaReviews: true,
```

Comment:
for complete status this should be false right? We do not want then to edit review note when the PIA is complete?

Reply:
This does not affect editReviewNote. This is for _showing_ the review 'blocks'.